### PR TITLE
Feat/master/publish to pypi

### PR DESCRIPTION
**Pull request by anestos**

### CODE REVIEW

File: flask_of_oil/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+__version__ = '0.0.1'
```
Comment (travisspencer) [inline flask_of_oil/__init__.py]: not 1.0.0?

**Review by travisspencer — APPROVED** [review]

File: flask_of_oil/opaque_validator.py [hunk 2]
```diff
@@ -142,4 +142,5 @@ def validate(self, token):
 
         return {"subject": introspect_response['sub'],
                 "scope": introspect_response['scope'],
+                "cnf": introspect_response.get("cnf", ""),
```
Comment (travisspencer) [inline flask_of_oil/opaque_validator.py]: good

File: flask_of_oil/__init__.py (hunk 1 above)
Comment (anestos) [reply flask_of_oil/__init__.py]: will go to 1.0.0 when merged, right now i am testing the publishing part and will use 0.0.1,0.0.2 etc..
after publishing, the version cannot be re-published in pypi, just deleted, so we have to make it right

File: flask_of_oil/opaque_validator.py (hunk 2 above)
Comment (anestos) [reply flask_of_oil/opaque_validator.py]: Maybe we should revert this as it doesn't make any difference right now. We should find a more generic approach to send back all claims to the request.

We only use the `active`, `scope` and `subject` from opaque_validator.validate (and the jwt_validator.validate method) in oauth_filter.filter.

`active` is checked for True/False, `scope` is used to authorize the request comparing with the configured scopes and `subject` is saved into g.user

Comment (travisspencer) [reply flask_of_oil/opaque_validator.py]: ya, something generic like we did in the Java API when we updated it for the licensing service. That's what we need.

**Pull request merged**